fate/src/main/java/org/apache/accumulo/fate/AdminUtil.java

```diff
@@ -110,25 +110,36 @@ public String getTop() {
   public static class FateStatus {
 
     private final List<TransactionStatus> transactions;
-    private final Map<Long,List<String>> danglingHeldLocks;
-    private final Map<Long,List<String>> danglingWaitingLocks;
+    private final Map<String,List<String>> danglingHeldLocks;
+    private final Map<String,List<String>> danglingWaitingLocks;
+
+    private static Map<String,List<String>> convert(Map<Long,List<String>> danglocks) {
+      if (danglocks.isEmpty()) {
+        return Collections.emptyMap();
+      }
+
+      Map<String,List<String>> ret = new HashMap<>();
+      for (Entry<Long,List<String>> entry : danglocks.entrySet()) {
+        ret.put(String.format("%016x", entry.getKey()), Collections.unmodifiableList(entry.getValue()));
```

Comment:
That's fine. This is more clear in the previous code when the formatting was in the log message. As a private utility method, it's far less clear how the formatting is used, or why it matters. It's not clear, for example, that this is just pre-formatting the object for future logging. The method is making assumptions about the caller's behavior.  It's probably fine, but it'd help if there were a comment explaining it.